features/cellular/framework/AT/AT_CellularNetwork.cpp

```diff
@@ -61,6 +61,7 @@ AT_CellularNetwork::~AT_CellularNetwork()
     }
 
     _at.remove_urc_handler("NO CARRIER", callback(this, &AT_CellularNetwork::urc_no_carrier));
+    _at.remove_urc_handler("+CGEV:", callback(this, &AT_CellularNetwork::urc_cgev));
```

Comment:
Should also be removed in `disconnect()`

Reply:
done